#!/usr/bin/env python
PACKAGE = "ilcc2"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#============================================================ initial value | lower bound | higher bound
gen.add("roll",     double_t,   0, "prior param Lidar to cam (degree)",  -90,   -100,   -85)
gen.add("pitch",    double_t,   0, "prior param Lidar to cam (degree)",    0,   -5,     5)
gen.add("yaw",      double_t,   0, "prior param Lidar to cam (degree)", -180,   -180,   180)
gen.add("pos_x",    double_t,   0, "prior param Lidar to cam (m)",         0,   -1,     1)
gen.add("pos_y",    double_t,   0, "prior param Lidar to cam (m)",         0,   -1,     1)
gen.add("pos_z",    double_t,   0, "prior param Lidar to cam (m)",         0,   -1,     1)

exit(gen.generate(PACKAGE, "ilcc2", "Locator"))
